miio/tests/test_g1vacuum.py

```diff
@@ -0,0 +1,85 @@
+from unittest import TestCase
+
+import pytest
+
+from miio.g1vacuum import (
+    G1ChargeState,
+    G1FanSpeed,
+    G1MopState,
+    G1State,
+    G1Vacuum,
+    G1VacuumMode,
+    G1WaterLevel,
+)
+
+from .dummies import DummyMiotDevice
+
+_INITIAL_STATE = {
```

Comment:
Use the example response as shown inside the status class for tests, the idea is to verify that this information is parsed correctly by the status class.

Reply:
Right now, I don't quite understand the whole testing  process and what I should change here. Unfortunatly my time slot for this project is really minimal.

Reply:
Ah, no worries! Considering that there'll be some flux in the library (including the tests) to make it have common interfaces, and that there is an example output for the status container, let's just merge this without tests for now.

So please just remove `test_g1vacuum.py` from this pull request and it's ready to be merged :-)